community/kernel/src/main/java/org/neo4j/storageengine/api/CommandReaderFactory.java

```diff
@@ -30,9 +30,7 @@
      * introduce its own scheme.
      *
      * @param version log entry version. Versions are typically 0 or negative numbers.
-     * @param legacyHeaderVersion legacy log header version for breaking tie between multiple versions
-     * of value {@code 0}. Will go away when 1.9 format becomes unsupported.
      * @return {@link CommandReader} for reading commands of that version.
      */
-    CommandReader byVersion( byte version, byte legacyHeaderVersion );
+    CommandReader byVersion( byte version );
```

Comment:
Not sure, the interface is part of the storageengine api, the other one is an internal implementation. Should we really make the internal class part of the api?